db/seeds.rb

```diff
@@ -11,7 +11,32 @@
               posting_date: rand(1...10).days.ago,
               description: Faker::Lorem.sentence(3),
               location: "#{Faker::Address.city}, #{Faker::Address.state}",
-              company_id: rand(1...20),
+              company_id: rand(1...10),
               source: Faker::Name.name
             )
 end
+  puts "Jobs Created"
```

Comment:
Inconsistent indentation detected.<br>Do not write to stdout. Use Rails' logger if you want to log.